frontend/src/app/app.module.ts

```diff
@@ -13,6 +13,8 @@ import { DataModule } from './data/data.module';
 
 import { AppConfigModule } from './app-config.module';
 import { AppRoutingModule } from './app-routing.module';
+import { EventsModule } from "./events/events.module";
+import { GeneralModule } from "./general/general.module";
 
```

Comment:
general module gives an error
ERROR in src/app/app.module.ts(17,31): error TS2307: Cannot find module './general/general.module'.
removing general module fixes the error

Reply:
Thanks I will fix it, or you can too

Reply:
Done